pkgs/applications/networking/qv2ray/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, qt5
+, cmake
+, clang
+, grpc
+, protobuf
+, openssl
+, pkgconfig
+, c-ares
+, abseil-cpp
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qv2ray";
+  version = "2.5.0";
+
+  src = fetchFromGitHub {
+    owner = "Qv2ray";
+    repo = "Qv2ray";
+    rev = "v${version}";
+    sha256 = "12x72rsd5iaj06qmma8x6g7syqaiixnfb37f3ppx8x00c77s78va";
+    fetchSubmodules = true;
+  };
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DQV2RAY_DISABLE_AUTO_UPDATE=on"
+    "-DQV2RAY_TRANSLATION_PATH=$(out)/share/qv2ray/lang"
+  ];
+
+  preConfigure = ''
+    export _QV2RAY_BUILD_INFO_="Qv2ray Nixpkgs"
+    export _QV2RAY_BUILD_EXTRA_INFO_="(Nixpkgs build) $(uname -a | cut -d " " -f3,13)"
+  '';
+
+  installPhase = ''
+    cmake --install .
+  '';
```

Comment:
are you sure this is not the default?

Reply:
Yep that seems to be the default. I was not sure cuz I tried to grep "cmake --install" and didn't find anything.